lib/server.js

```diff
@@ -102,12 +113,15 @@ async function readConfiguration() {
 async function startHttpServer(config) {
   const { host, port } = config.listen.http;
 
-  try {
-    await http.createServer(app(config)).listen(port, host);
-    logger.info(`[${process.pid}] HTTP: listening on port ${port}`);
-  } catch (e) {
-    throw new Error(`unable to start start http server: ${e.message}`);
-  }
+  return new Promise((resolve, reject) => {
```

Comment:
btw: I think that `await http.createServer(app(config)).listen(port, host)` never worked.... http.createServer is not promisified.